fdbserver/masterserver.actor.cpp

```diff
@@ -1925,7 +1940,7 @@ ACTOR Future<Void> masterCore(Reference<MasterData> self) {
 
 	// resize the previous commit version vector to the number of tlogs.
 	int numLogs = self->dbInfo->get().logSystemConfig.numLogs();
-	self->tpcvVector.resize(1 + numLogs, invalidVersion);
+	self->tpcvVector.resize(1 + numLogs, 0);
```

Comment:
As we discussed, it may require a whole lot of effort to identify the correct set of initial values here. So the plan is to special case this in the tlog?

A comment about why the vector is getting initialized with zero version values would be nice.

Reply:
In unmodified fdb I think prevVersion is created with a value of 0 (what we get on very first transaction), and is set to `self->lastEpochEnd;` on recovery. I'll add a comment here and see if we can use logic similar to [this](https://github.com/apple/foundationdb/blob/8ecff8c7304f71be30f3ed6f9b51263c7c2dcc4a/fdbserver/masterserver.actor.cpp#L1156) for the tpcvVector.

Reply:
I think we will need something like an array of `lastEpochEnd` per tlog. This will probably be done when we implement recovery.